src/Version.php

```diff
@@ -1,226 +1,162 @@
 <?php
+
 /**
- * Zend Framework (http://framework.zend.com/)
+ * Zend Framework (http://framework.zend.com/).
  *
  * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ *
  * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
  * @license   http://framework.zend.com/license/new-bsd New BSD License
  */
-
 namespace Zend\Version;
 
-use Zend\Http;
-use Zend\Json\Json;
-
 /**
- * Class to store and retrieve the version of Zend Framework.
+ * Encapsulates semantic version information.
  */
-final class Version
+class Version
 {
     /**
-     * Zend Framework version identification - see compareVersion()
+     * The current version alias.
+     *
+     * @var string
+     * @deprecated 3.0.0  use Zend\Version\CURRENT instead
      */
-    const VERSION = '2.6.0dev';
+    const VERSION = CURRENT;
```

Comment:
If we did that, we'd need to load the class just to have access to a 'global' constant.  Instead, this is what I was going for:
```php
use const Zend\Version\CURRENT as ZF_VERSION;
```
See the PR examples.